corehq/apps/userreports/reports/view.py

```diff
@@ -232,7 +233,11 @@ def request_user(self):
     @property
     @memoized
     def filter_values(self):
-        return get_filter_values(self.filters, self.request_dict, user=self.request_user)
+        filters = get_filter_values(self.filters, self.request_dict, user=self.request_user)
+        for key in filters:
+            if isinstance(filters[key], DateSpan) and filters[key].inclusive:
```

Comment:
Hey @Rohit25negi this is not the right place to make this change, each filters have their own processing functions, so it will be good to move this out there. I can show you this offline.

Reply:
you meant this https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/reports_core/filters.py#L109?
If not we can take it offline :)